pkg/noun/events.c

```diff
@@ -296,24 +296,29 @@ u3e_fault(u3_post low_p, u3_post hig_p, u3_post off_p)
       fprintf(stderr, "loom: strange guard (%d)\r\n", pag_w);
       return u3e_flaw_sham;
     }
+
+    if ( u3P.dit_w[blk_w] & (1 << bit_w) ) {
+      fprintf(stderr, "loom: strange guard page (%d): %x\r\n", pag_w, off_p);
+      return u3e_flaw_sham;
+    }
+
+    u3P.dit_w[blk_w] |= (1 << bit_w);
```

Comment:
Clearing the bit here is redundant, it's an error if it's set.

Reply:
Good catch, that refactor was wrong.  I tested that this version failed whenever we move the guard page, and deleting this made it work in the same case.

Reply:
Glad to see CI caught that as well: https://github.com/urbit/vere/actions/runs/5326673779